pkg/repoowners/repoowners.go

```diff
@@ -77,6 +77,7 @@ type FullConfig struct {
 type scmProviderClient interface {
 	ListCollaborators(org, repo string) ([]scm.User, error)
 	GetRef(org, repo, ref string) (string, error)
+	GetFile(string, string, string, string) ([]byte, error)
```

Comment:
Add a more descriptive name to the args instead of (string, string, string, string), just like ListCollaborators and GetRef ...

Reply:
OK. I just copied from pkg/scmprovider/client.go:31 ...